src/DelegationWall.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.17;
+
+/// @title DelegationWall - Release 1, Build 1
+/// @author Aragon Association - 2024
+/// @notice A smart contract where any wallet can register its own libsodium public key for encryption purposes
+contract DelegationWall {
+    struct Candidate {
```

Comment:
I think it will be better posting something in IPFS rather than this.
Cheaper and more flexible

Reply:
Renamed

Reply:
But then, the social is not needed anymore, isn't it? We could just remove that

Reply:
Done